### PR TITLE
[bitnami/wordpress] Added SSL support

### PR DESCRIPTION
**Pull request by FerronN**
Signed-off-by: Ferron Nijland <fernijland@hotmail.com>

Solves #17802

### CODE REVIEW

**Comment by FerronN** [comment]
It looks like this works when not persisting the `wp-config.php`. 
I can't find why (at this time), but `db_flags="$(wordpress_conf_get "MYSQL_CLIENT_FLAGS")"` doesn't return the defined value in `wp-config.php` which is correct. It returns 2048...

File: bitnami/wordpress/6/debian-11/rootfs/opt/bitnami/scripts/libwordpress.sh [hunk 1]
```diff
@@ -565,8 +587,11 @@ wordpress_wait_for_mysql_connection() {
     local -r db_name="${3:?missing database name}"
     local -r db_user="${4:?missing database user}"
     local -r db_pass="${5:-}"
+    local -r db_options="${6:-}"
     check_mysql_connection() {
-        echo "SELECT 1" | mysql_remote_execute "$db_host" "$db_port" "$db_name" "$db_user" "$db_pass"
+        #Remove test first line
+        echo "SELECT 1" "$db_host" "$db_port" "$db_name" "$db_user" "$db_pass" "$db_options"
```
Comment (Mauraza) [inline bitnami/wordpress/6/debian-11/rootfs/opt/bitnami/scripts/libwordpress.sh:593]: I think this line isn't necessary, could you check it?

**Comment by Mauraza** [comment]
Hi @FerronN,

I see that the PR is un Draft, could you ping me when it's ready?

**Comment by FerronN** [comment]
@Mauraza I will. I'm currently in the process of solving the bug with the wp-cli team.

**Comment by FerronN** [comment]
@Mauraza It seem like the WP-CLI has no clue for now regarding the issue mentioned earlier. 
So I've made a work around. For this to work the: `WORDPRESS_ENABLE_DATABASE_SSL` and `WORDPRESS_VERIFY_DATABASE_SSL` always need to be present. It will be saved into wp-config (for when the workaround can be fixed), but it won't get the value from the wp-config.

**Comment by Mauraza** [comment]
Hi @FerronN,

I going to check all. Could you sing the commits?
You can follow this instruction https://github.com/bitnami/containers/pull/19204/checks?check_run_id=11219797719

**Pull request closed without merging**